src/main/java/seedu/address/logic/commands/ProfCommand.java

```diff
@@ -53,7 +53,7 @@ public CommandResult execute(Model model) throws CommandException {
             try {
                 model.addPerson(person);
             } catch (DuplicatePersonException e) {
-                throw new CommandException("Error: Professor already in contacts");
+                // throw new CommandException("Error: Professor already in contacts");
             }
         }
 
```

Comment:
seems to work, on my local machine, though im not sure whats causing the errors!